plugins/lsp/language-servers/nil-ls-settings.nix

```
@@ -0,0 +1,58 @@
+{ lib, helpers }:
+# Options:
+#  - https://github.com/oxalica/nil/blob/main/docs/configuration.md?plain=1#
+with lib;
+{
+  formatting = {
+    command = helpers.defaultNullOpts.mkNullable (types.listOf types.str) "null" ''
```

```suggestion
    command = helpers.defaultNullOpts.mkListOf types.str "null" ''
```
Use `mkListOf` where possible.